dracut/99switch-root/initrd-parse-etc-override.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPost=-/bin/sh -c "[ -f /sys/fs/cgroup/cgroup.subtree_control ] && [ -f /sysroot/etc/flatcar-cgroupv1 ] && echo INIT=/sbin/init.wrapper >>/etc/switch-root.env"
```

Comment:
`/etc/switch-root.env` gets created here? Then I would think that ` >/etc…` is better than appending. I also would like to have `INIT` be defined in the other case because in `overwrite.conf` it gets used in both cases.

Reply:
Yes `switch-root.env` is created here, so the ">" is better, will switch. I'm not sure about defining in both cases though, it really needs to be empty for the other cases, and that's why the EnvironmentFile directive uses `=-` to allow the file to be missing

Reply:
I found no docs on the usage of undefined variables. If it works, fine, but we could also set it with `Environment="INIT="` to define it and it will then be overwritten by `EnvironmentFile` (this gets evaluated last)